samples/OpenIdConnect.AzureAdSample/project.json

```diff
@@ -8,7 +8,7 @@
     "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.0.0-*",
     "Microsoft.Extensions.Configuration.UserSecrets": "1.0.0-*",
     "Microsoft.Extensions.Logging.Console": "1.0.0-*",
-    "Microsoft.IdentityModel.Clients.ActiveDirectory": "3.9.302261508-alpha"
+    "Microsoft.IdentityModel.Clients.ActiveDirectory": "3.11.305310302-alpha"
   },
   "frameworks": {
     "net451": { },
```

Comment:
Ah yeah, I need glasses. 

> Le 1 juin 2016 à 15:49, Chris R <notifications@github.com> a écrit :
> 
> In samples/OpenIdConnect.AzureAdSample/project.json:
> 
> > @@ -8,7 +8,7 @@
> >      "Microsoft.Extensions.Configuration.EnvironmentVariables": "1.0.0-*",
> >      "Microsoft.Extensions.Configuration.UserSecrets": "1.0.0-*",
> >      "Microsoft.Extensions.Logging.Console": "1.0.0-*",
> > -    "Microsoft.IdentityModel.Clients.ActiveDirectory": "3.9.302261508-alpha"
> > +    "Microsoft.IdentityModel.Clients.ActiveDirectory": "3.11.305310302-alpha"
> >    },
> >    "frameworks": {
> >      "net451": { },
> no, the major, minor, and patch are all ints, it only gets into strict alphabetical in the prerelease tag.
> 
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub, or mute the thread.
>